web-api/src/users/updateUserPendingEmailLambda.js

```diff
@@ -0,0 +1,17 @@
+const { genericHandler } = require('../genericHandler');
+
+/**
+ * updates the user pending email
+ *
+ * @param {object} event the AWS event object
+ * @returns {Promise<*|undefined>} the api gateway response object containing the statusCode, body, and headers
+ */
+exports.updateUserPendingEmailLambda = event =>
+  genericHandler(event, async ({ applicationContext }) => {
+    return await applicationContext
+      .getUseCases()
+      .updateUserPendingEmailInteractor({
+        applicationContext,
+        ...JSON.parse(event.body),
```

Comment:
Can we only send this what it needs instead of what it gets?

Reply:
When the contract between the client proxy and the interactor changes, this will be one layer which need not change.  I think it is better to simply pass along the entire body, no harm done.